pkg/disruptors/pod_test.go

```diff
@@ -123,8 +124,10 @@ func Test_PodHTTPFaultInjection(t *testing.T) {
 					},
 				},
 			},
-			target:      buildPodWithPort("my-app-pod", "http", 80),
-			expectedCmd: "xk6-disruptor-agent http -d 60s -t 80 -r 0.1 -e 500 -b {\"error\": 500}",
+			target: buildPodWithPort("my-app-pod", "http", 80),
+			// Lack of quotes on the -b arg is okay. Argument is returned as []string, and it is compared as string
```

Comment:
It confused me that we were considering `-b {\"error\": 500}` a valid cmdline as, at first glance, there's a space in `: 500` which should make it invalid. Then, I realized that what we are using the string representation only for testing, the real argument doesn't look like that (it is returned as `[]string`). I wanted to save future me the same moment of panic with the comment, but maybe I didn't get the wording right.

Reply:
Maybe we should then fix the test (not in this here) and make the 'expectedCmd` to be `[]string` to match the actual type. I would add this as a comment, instead of explaining why it is broken but still works.

Reply:
LGTM, reworded as a `TODO`.